### DISCUSSION TITLE
Domotixz IDX funktioniert mit 14-0-0 nicht mehr

### DISCUSSION THREAD

**Discussion by mafrei2019** [arendst/Tasmota#21438] [Support request]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by arendst** [comment]
DzIdx works great:

```
11:22:24.072 SRC: MQTT
11:22:24.073 CMD: Grp 0, Cmd 'POWER', Idx 1, Len 299, Pld 1, Data '1'
11:22:24.121 MQT: stat/powr320d3/RESULT = {"POWER":"ON"}
11:22:24.123 MQT: stat/powr320d3/POWER = ON
11:22:25.362 CFG: Saved, Count 1286, Bytes 4096
11:22:32.872 CSE: Overload
11:22:37.224 CMD: dzidx
11:22:37.225 SRC: WebConsole from 192.168.2.1
11:22:37.226 CMD: Grp 0, Cmd 'DZIDX', Idx 1, Len 0, Pld -99, Data ''
11:22:37.231 MQT: stat/powr320d3/RESULT = {"DzIdx1":403}
11:22:50.833 CMD: dzidx0 0
11:22:50.834 SRC: WebConsole from 192.168.2.1
11:22:50.835 CMD: Grp 0, Cmd 'DZIDX', Idx 0, Len 1, Pld 0, Data '0'
11:22:50.837 MQT: Unsubscribe from domoticz/out/#
11:22:50.845 MQT: stat/powr320d3/RESULT = {"DzIdx0":0}
11:22:51.089 CFG: Saved, Count 1287, Bytes 4096
11:22:58.600 CMD: dzidx
11:22:58.601 SRC: WebConsole from 192.168.2.1
11:22:58.603 CMD: Grp 0, Cmd 'DZIDX', Idx 1, Len 0, Pld -99, Data ''
11:22:58.608 MQT: stat/powr320d3/RESULT = {"DzIdx1":0}
11:23:08.808 CMD: dzidx1 403
11:23:08.809 SRC: WebConsole from 192.168.2.1
11:23:08.810 CMD: Grp 0, Cmd 'DZIDX', Idx 1, Len 3, Pld 403, Data '403'
11:23:08.813 MQT: Subscribe to domoticz/out/#
11:23:08.820 MQT: stat/powr320d3/RESULT = {"DzIdx1":403}
11:23:09.019 CFG: Saved, Count 1288, Bytes 4096
```

If you encounter issues then fill in the requested information from the initial issue and we might find out more....

**Comment by mafrei2019** [comment]
Ich habe es nochmal probiert. Schalte in in Domo eine Gerät welches als Info nur die IDX (mqtt) eingetragen hat, funktioniert es bein14-0-0 nicht aber bei 13-4-0 ohne Probleme. Somit funktionieren auch alle meine Blockly nicht mehr.
